### PR TITLE
New api

### PR DESCRIPTION
**Pull request by githubmo**
##What

Set of new JSONs based on the idea of the new API structure.

### How to review

- Make sure the JSON files in the stub represent the data the backend wants to provide and the front-end wants to consume
- Make sure the changes to the handlers are not to erroneous 
- Run `make debug` and make sure you can access the resources you require

### Who can review

@NeilMadden @fullstackforger

### CODE REVIEW

**Comment by githubmo** [comment]
Do not merge it yet until @fullstackforger is happy to merge.

File: handlers/dataset/handler.go
Comment (NeilMadden) [inline handlers/dataset/handler.go:21]: Proper log statements... ;-)

**Review by NeilMadden — APPROVED** [review]
Looks good to me. Love the log messages! 👍

**Pull request merged**
